pkg/scalers/artemis_scaler.go

```diff
@@ -33,6 +34,8 @@ type artemisMetadata struct {
 	queueLength        int
 }
 
+//revive:enable:var-naming
```

Comment:
I like it!

Reply:
@turbaszek @kashifsoofi I haven't had a time to look at revive docs yet, is there a TLDR explanation for this particular comment, please? Thanks!

Reply:
Basically it allows you to ignore some warnings on code-line leve. In case you want to disable only a particular rule, you can use:
```go
//revive:disable:unexported-return
func Public() private {
  return private
}
//revive:enable:unexported-return
```
This way, revive will not warn you for that you're returning an object of an unexported type, from an exported function.

Details: https://github.com/mgechev/revive#comment-directives

Reply:
Gotcha, thanks. I'd like to avoid spreading this kind of markers across the code. But as an exception for this particular case it is ok.

Reply:
@zroubalik I totally agree that we should limit such markers. We should use them only in cases like this one where linting may introduce breaking change.